app/components/post_textbox/post_textbox.js

```diff
@@ -79,7 +79,7 @@ export default class PostTextbox extends PureComponent {
     }
 
     componentWillReceiveProps(nextProps) {
-        if (nextProps.channelId !== this.props.channelId || nextProps.rootId !== this.props.rootId) {
+        if (nextProps.channelId !== this.props.channelId || nextProps.rootId !== this.props.rootId || nextProps.value !== this.state.value) {
```

Comment:
This is breaking https://github.com/mattermost/mattermost-mobile/pull/1138 which coincidentally has a comment about breaking this feature

@enahum Have we confirmed the performance benefits of not updating the value of the textbox right away? We should come up with a way of consolidating these two changes if that's the case. Maybe have something we can trigger when clicking on the username to force the textbox to pick up new changes to the saved value

Reply:
Oh yes is considerably slow as it will re-compute everything, maybe this can be fix by emitting an event?

Reply:
@enahum I'll implement something with the event emitter